app/components/dashboard/tour/menu_id/location/GoogleMaps.js

```diff
@@ -0,0 +1,46 @@
+import { Loader } from "@googlemaps/js-api-loader";
+import { useEffect, useRef } from "react";
+
+export default function GoogleMaps() {
+  const mapRef = useRef(null);
+
+  useEffect(() => {
+    const initMap = async () => {
+      const loader = new Loader({
+        apiKey: process.env.NEXT_PUBLIC_GOOGLE_MAPS_API_KEY,
+        version: "weekly",
+      });
+
+      try {
+        const google = await loader.load();
+        const Map = new google.maps.Map(mapRef.current, {
+          center: { lat: 34.4905458, lng: -90.3871189 },
+          zoom: 10,
+          mapId: "MY_NEXTJS_MAPID",
+        });
+
+        new google.maps.Marker({
+          position: { lat: 34.4905458, lng: -90.3871189 },
```

Comment:
these should be stored in an enum (each entry being a map instead of a string, and the key can be a name for the location)

Reply:
nice! thank you so much
done